R/helpers-misc.R

```diff
@@ -61,15 +61,17 @@ everything <- function(vars = NULL) {
 #' @export
 #' @param offset Set it to `n` to select the nth var from the end.
 last_col <- function(offset = 0L, vars = NULL) {
-  stopifnot(is_integerish(offset))
+  if (!is_integerish(offset, n = 1)) {
+    abort(glue("`offset` must be a single integer."))
```

Comment:
Here and below: `cli_abort()` with `.arg` style.

Reply:
Can also check for finiteness with `finite = TRUE`. rlang does not export this helper yet but it could be added to `utils.R`:

```r
is_number <- function(x) {
  is_integerish(x, n = 1, finite = TRUE)
}
```